src/components/subComps/educationArtCard/EducationArtCard.tsx

```diff
@@ -10,50 +9,45 @@ import { EducationArtCardProps } from '../../../utils/interfaces';
 import BookmarkBorderIcon from '@mui/icons-material/BookmarkBorder';
 import BookmarkAddedIcon from '@mui/icons-material/BookmarkAdded';
 
-
-const EducationArtCard: React.FC<EducationArtCardProps> = ({ title, tagline, id, handleClick, handleSaves, savedArticles, isSaved }) => {
-
-    return (
-        <Grid item >
-        <Card
+const EducationArtCard: React.FC<EducationArtCardProps> = ({
```

Comment:
I do like this way of defining using TypeScript, I believe we can still use our own interface props within the file. This will make it easier to make changes to props down the road.